Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:current-alpine
```

Comment:
How will we handle updates to this image? If we e.g. want to bump the Node.js version.

Reply:
This should (should?) always pull the latest for both Node and Alpine.

https://hub.docker.com/_/node

Reply:
It'd be safer if we specified version 14 here as we did for Mobile.